nuxeo-runtime/nuxeo-runtime/src/main/java/org/nuxeo/runtime/RuntimeExtension.java

```diff
@@ -44,4 +44,18 @@
      */
     void stop();
 
+    /**
+     * Resumes the runtime extension.
+     *
+     * @since 9.2
+     */
+    void resume();
+
+    /**
+     * Put the extension in standby mode.
+     *
+     * @since 9.2
+     */
+    void standby();
```

Comment:
Why is there no `Instant` deadline for this? Especially since no one implements it yet it's not a big constraint.